PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.sqlserver.common;
+
+import com.microsoft.azure.toolkit.intellij.common.AzureComboBox;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.sqlserver.model.SqlDatabaseEntity;
+import com.microsoft.azure.toolkit.lib.sqlserver.service.ISqlServer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class SqlServerDatabaseComboBox extends AzureComboBox<SqlDatabaseEntity> {
```

Comment:
I don't see any valuable thing in this class than DatabaseComboBox

Reply:
added todo to track for refacor.